src/transformers/pipelines/zero_shot_image_classification.py

```diff
@@ -17,18 +18,16 @@
     from ..image_utils import load_image
 
 if is_torch_available():
-    import torch
+    pass
```

Comment:
to remove?

Reply:
Actually I think we can remove thanks to @yessenzhar